src/CalcViewModel/DataLoaders/UnitConverterDataLoader.cpp

```diff
@@ -120,6 +120,12 @@ void UnitConverterDataLoader::LoadData()
 
                 for (auto itr = unitConversions.begin(); itr != unitConversions.end(); ++itr)
```

Comment:
There's an opportunity to improve the readability of the code here with:
```suggestion
                 for (const auto&[id, conversionFactor] : unitConversions)
```
Then replace `itr->first` with `id` and `itr->second` with `conversionFactor`.